src/main/resources/liquibase/crop_changelog/17_1_0.xml

```diff
@@ -27,4 +27,20 @@
 		<comment>Remove personlist table</comment>
 		<dropTable tableName="personlist"></dropTable>
 	</changeSet>
+
+	<changeSet author="nahuel" id="v17.1.0-4">
+        <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				select count(1) from sequence where sequence_name = 'nd_experimentprop'
+			</sqlCheck>
+        </preConditions>
+        <comment>
+            Create sequence entry for nd_experimentprop table generator
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+			insert into sequence(sequence_name, sequence_value)
```

Comment:
Question, what was the 1) rationale and has been 2)  the effect of using sequence entry for table generation for `nd_experimentprop`?